### PR TITLE
[trivial] Refactor / cleanup some code parts

### LINKED ISSUE #45: Minor code cleanup
In the course of implementing https://github.com/els0r/goProbe/issues/83 a few minor refactor / cleanup tasks came up. These are tracked here.

### PR DESCRIPTION
**Pull request by fako1024**
Closes #45

### CODE REVIEW

File: capture/afpacket/afring/tpacket.go [hunk 1]
```diff
@@ -231,3 +176,41 @@ func blockSizeTPacketAlign(x, blockSize int) (int, error) {
 
 	return 0, fmt.Errorf("no valid frame size found for capture length %d / block size %d", x, blockSize)
 }
+
```
Comment (fako1024) [inline capture/afpacket/afring/tpacket.go:179]: I've moved all currently unused methods down here and commented them (might be that we use some of them in the future, and looking up the required struct positions can be tedious, so I hope keeping commented code is acceptable in this situation...

**Comment by fako1024** [comment]
@els0r Trivial refactor / cleanup, feel free to approve at your convenience. TY!

**Comment by fako1024** [comment]
@els0r Friendly ping... :pray:

**Review by els0r — APPROVED** [review]

**Comment by els0r** [comment]
I keep missing the slimcap stuff. Sry

**Comment by fako1024** [comment]
> I keep missing the slimcap stuff. Sry

No worries, thanks man!

**Pull request merged**